link/res/values/strings.xml

```diff
@@ -26,6 +26,7 @@
     <string name="wallet_expand_accessibility">Change selection</string>
     <string name="wallet_default">Default</string>
     <string name="wallet_update_card">Update card</string>
+    <string name="wallet_set_as_default">Set as default</string>
```

Comment:
Seems like we have lokalise translations for this string, lets add them? Feel free to add them in a separate PR: https://app.lokalise.com/project/747824695e51bc2f4aa912.89576472/?view=multi&search=Set%20as%20default

Reply:
Yep, adding the translations for this string (and all the other ones) in a separate pull request.